gruntwork-install

```diff
@@ -282,8 +284,9 @@ function install_script_module {
         shift
         ;;
       --module-param)
-        local readonly param=$(convert_module_params_format "$2")
-        module_params+=("$param")
+        local -a param=()
+        mapfile -t param <<<"$(convert_module_params_format "$2")"
```

Comment:
Ensure that we ingest the params as separate params and don't get tripped up by whitespace.

Reply:
Hm, one downside is `mapfile` isn't available in OS X by default... Not sure about the Linux distro's we support. Is there an alternative, more portable way to do this?

Reply:
I tried a different approach in the next commit. Take a look and let me know what you thing.